src/Resources/views/layout.html.twig

```diff
@@ -18,8 +18,8 @@ file that was distributed with this source code.
             </div>
 
             {% block sonata_page_breadcrumb %}
-                <div class="row">
-                    {{ sonata_block_render_event('breadcrumb', { 'context': 'page', 'current_uri': app.request.requestUri }) }}
+                <div class="row page-breadcrumb">
+                    {{ sonata_block_render_event('breadcrumb', { 'context': 'sonata.page.block.breadcrumb', 'current_uri': app.request.requestUri }) }}
```

Comment:
do we know why this context changed? should we revert something somewhere?

Also if this code changes is there any docs to update?

Reply:
check this comment please : https://github.com/sonata-project/SonataPageBundle/pull/1564#issuecomment-1222008738

Reply:
But this bug was introduced here: https://github.com/sonata-project/SonataPageBundle/pull/1483/files#diff-f0461df8f4a53195f2b565ad37746df1e8076ff748edb18c3e2689581f2db4caR87

I guess he was just following how the stuffs were made in others parts, But in 3.x it was using `page`

and when added this method, it was changed!

I choose to use `sonata.page.block.breadcrumb` because it's more explainable then `page`, But yes, It's a BC!

because if someone is using  `sonata_block_render_event` passing page, it won't work!

I could back to `page`, But I guess it's not the correct correct for  `breadcrumb`, `sonata.page.block.breadcrumb` make more sense!

Reply:
I made a mistake there, imo it should be page

Reply:
well we can back this for page :), 

I'll open a PR for this ;)

Reply:
Yes if you can open a PR to fix the things I mentioned it would be great :) thanks!

Reply:
I opened https://github.com/sonata-project/SonataPageBundle/pull/1570